src/icons/general/IconAddTags.js

```diff
@@ -2,30 +2,33 @@
 import * as React from 'react';
 
 import AccessibleSVG from '../accessible-svg';
+import { bdlGray } from '../../styles/variables';
 
 type Props = {
     className?: string,
     color?: string,
     height?: number,
+    strokeWidth?: number,
     /** A text-only string describing the icon if it's not purely decorative for accessibility */
     title?: string | React.Element<any>,
     width?: number,
 };
 
-const IconAddTags = ({ className = '', height = 15, color = '#444', title, width = 15 }: Props) => (
+const IconAddTags = ({ className = '', height = 32, color = bdlGray, title, strokeWidth = 2, width = 32 }: Props) => (
     <AccessibleSVG
-        className={`icon-add-tags ${className}`}
+        className={`bdl-iconAddTags ${className}`}
```

Comment:
I think we've been using PascalCase for other icons, so `bdl-IconAddTags`.

Reply:
good catch